priv/module_static/lib/binary_tree.js

```diff
@@ -0,0 +1,90 @@
+// tree.js: Binary Tree abstraction for M5
```

Comment:
Having libraries explicitly included here will violate the "single source of truth" principle. We would have two versions and it wouldn't be clear what version is available in what situation.

Reply:
Okay, I have added the files under priv/modules to .gitignore.